packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -29,31 +28,58 @@
 			"maxNodes": 9
 		}
 	},
+
 	"paramInformation": [
 		{
 			"#": "1",
 			"label": "Sw1 Switch Type",
-			"description": "0",
+			"description": "This parameter defines how the Device should treat the switch (which type) connected to the SW (SW1) terminal.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 2,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "mono-stable switch type (push button)",
```

```suggestion
					"label": "Mono-stable (push button)",
```
Option labels should be **Sentence case**, also we try to prevent repetition between these and the parameter label.
https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=options

Likewise for the other ones.